test_utils/src/lib.rs

```diff
@@ -476,7 +476,15 @@ pub fn start_holochain_instance<T: Into<String>>(
     let test_config = env::var("INTEGRATION_TEST_CONFIG").map(|test_config|{
             if test_config =="lib3h"
             {
-                TestNodeConfig::MemoryGhostEngine(vec![])
+                let bootstrap_nodes = env::var("BOOTSTRAP_NODES").map(|bootstrap_config|{
+                    
+                    //https://bootstrap1,http://boostrap2 in the env config to set up before tests
```

Comment:
Is it intentional that one of these links is `https` and the other isn't?

Reply:
oh it was just a problem in the commenting. Mostly just parses a string to a url, doesn't really matter :)